src/main/scala/com/gu/mobile/notifications/football/Streams.scala

```diff
@@ -77,14 +80,10 @@ trait NotificationResponseStream extends NotificationsClient with Logging {
     * @param notifications The notifications to send
     * @return The stream
     */
-  def getNotificationResponses(notifications: Observable[Notification])(implicit executionContext: ExecutionContext): Observable[NotificationHistoryItem] =
+  def getNotificationResponses(notifications: Observable[GoalAlertPayload])(implicit executionContext: ExecutionContext): Observable[NotificationHistoryItem] =
     notifications flatMap { notification =>
-      Observable.defer(Observable.from(send(notification))).retry(retrySendNotifications) map { reply =>
-        NotificationSent(
-          new DateTime(),
-          notification,
-          reply
-        )
+      Observable.defer(Observable.from(send(notification))).retry(retrySendNotifications) map {
+        _ => NotificationSent(new DateTime(), notification)
```

Comment:
This is the way it always was.. I just changed the second parameter to be a goalAlertPayload object instead of a Notification.
But in any case I think it should be in the server's time zone